ITS#10235 slapo-nestgroup: silence extraneous register_at message

Provided by upstream, patch to be removed at next release.

--- openldap-2.6.8/servers/slapd/overlays/nestgroup.c.orig	2024-06-27 05:51:40.455445246 -0700
+++ openldap-2.6.8/servers/slapd/overlays/nestgroup.c	2024-06-27 05:52:27.444263483 -0700
@@ -876,7 +876,7 @@
 	"USAGE dSAOperation "			/* added; questioned */
 	"NO-USER-MODIFICATION " 		/* added */
 	"X-ORIGIN 'iPlanet Delegated Administrator' )",
-	&ad_memberOf, 0 );
+	&ad_memberOf, 1 );
 	if ( code && code != SLAP_SCHERR_ATTR_DUP ) {
 		Debug( LDAP_DEBUG_ANY,
 			"nestgroup_initialize: register_at (memberOf) failed\n" );
